merlin/datasets/entertainment/music_streaming/schema.json

```diff
@@ -72,7 +72,8 @@
     {
       "name": "item_genres",
       "valueCount": {
-        "min": "4"
+        "min": "4",
+        "max": "4"
```

Comment:
This change makes `Loader` to return `item_genres` as a multi-hot dense tensor rather than a ragged representation (__values, __offsets). I keep the `user_genres` feature as a ragged multi-hot feature, so that we test both cases